CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.0
```

Comment:
Sorry to be particular, but not sure this should be a major bump because we aren't changing the API of rubocop-govuk

Reply:
As in a people don't need to change how rubocop-govuk is configured in their repo.

Reply:
It does change the behaviour of repo - adding functionality of alerting about more linting errors. But isn't a breaking change in how you use rubocop-govuk